README.md

```diff
@@ -2,22 +2,30 @@
 [![Build Status](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-mscalendar/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-mscalendar)
 [![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-mscalendar/master.svg)](https://codecov.io/gh/mattermost/mattermost-plugin-mscalendar)
 
-# [Help Wanted](https://github.com/mattermost/mattermost-plugin-mscalendar/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3A%22up+for+grabs%22+label%3A%22help+wanted%22+sort%3Aupdated-desc)
+## Help Wanted tickets can be found [here](https://github.com/mattermost/mattermost-plugin-mscalendar/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3A%22up+for+grabs%22+label%3A%22help+wanted%22+sort%3Aupdated-desc)
+
+## Table of Contents
+1. [Overview](#overview)
+2. [Features](#features)
+3. [Configuration](#configuration)
+
+## Overview
 
 This plugin supports a two-way integration between Mattermost and Microsoft
 Outlook Calendar. For a stable production release, please download the latest
 version [in the Releases
 tab](https://github.com/mattermost/mattermost-plugin-mscalendar/releases) and
-follow [these instructions](#2-configuration) for install and configuration.
+follow [these instructions](#configuration) for install and configuration.
 
-## Table of Contents
+## Features
 
-- [1. Features](#1-features)
-- [2. Configuration](#2-configuration)
+- Daily summary of calendar events
+- Automatic user status synchronization into Mattermost
+- Accept or decline calendar event invites from Mattermost
 
-## 2. Configuration
+## Configuration
 
-### Step 1 Create Mattermost App Azure (Private or Enterprise MS account)
+### Step 1: Create Mattermost App Azure (Private or Enterprise MS account)
 
 - Sign into [portal.azure.com](www.portal.azure.com)
   - from the hamburger menu -> `Azure Active Directory`
```

Comment:
Maybe is not yet the time, but I think that adding screenshots would make things way clearer.

Reply:
Agree and adding a section for available slash commands.  @mickmister, up to you, since this PR is about bots and permissions.  Maybe create as a separate ticket? Even a HW..?

Reply:
I think the barrier to entry on developing on the plugin has been a bit high, so I'll take some time to do some updates like this.

Reply:
I've left the slash commands out, since I was focusing specifically on setting up the plugin in this PR.

Reply:
Maybe create an issue about adding and updating the Slash commands?